app/_src/kubernetes-ingress-controller/faq.md

```diff
@@ -2,16 +2,24 @@
 title: FAQs
 ---
 
-### Why endpoints and not services?
+{% if_version lte: 2.9.x %}
+### Why `Endpoint`s and not `Service`s?
```

Comment:
The upstream Kubernetes style guide and content are unfortunately inconsistent about this, but the majority of content does not code format the names of resources, e.g. [as seen on the Service DNS page](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#namespaces-of-services). Code formatting them ends up being a bit wonky because they still get used as parts of speech and need plural/possessive/etc. forms that break the formatting.

```suggestion
### Why Endpoints and not Services?
```
The  exception is when these are literal values in other fields, e.g. the Events page below where `Ingress` is the value of the `InvolvedObject` field.